tests/org.eclipse.n4js.lang.tests/src/org/eclipse/n4js/tests/scoping/InfiniteRecursionTest.xtend

```diff
@@ -164,9 +164,9 @@ class InfiniteRecursionTest {
 	@Ignore
 	@Test //(timeout = 3500)
 	def void testComputationFinishes_02() {
-		val scriptAsString = CharStreams.toString[|
-			new InputStreamReader(class.getResourceAsStream("InfiniteComputationTest_02.txt"))
-		]
+		val fileURL = class.getResource("InfiniteComputationTest_02.txt")
```

Comment:
Looks like you want to have a utiliy that does these three lines consistently. Also there is Resources.toString

Reply:
Thanks for the hint!